packages/icons/test/index.js

```diff
@@ -13,20 +16,13 @@ test.each(iconList)('renders %s', (key, Component) => {
 })
 
 describe('Icon', () => {
-  test('renders custom icon', () => {
-    const expected = TestRenderer.create(<AirplaneIcon />).toJSON()
-    const json = TestRenderer.create(<Icon name="Airplane" />).toJSON()
-    expect(json).toEqual(expected)
-  })
-
-  test('renders material design icon', () => {
-    const expected = TestRenderer.create(<AcIcon />).toJSON()
-    const json = TestRenderer.create(<Icon name="Ac" />).toJSON()
-    expect(json).toEqual(expected)
+  test('defaults title to name if not provided', () => {
+    const testRenderer = TestRenderer.create(<Icon name="Accessible" />)
+    const testInstance = testRenderer.root
+    expect(testInstance.children[0].props.title).toBe('Accessible')
   })
 
-  describe('SVG Icon Accessibility', () => {
-    // Test individual icons in './lib'
+  describe.skip('SVG Icon Accessibility', () => {
```

Comment:
skipping this test?

Reply:
I'll address this today. Thanks for a good catch!

Reply:
Addressed in https://github.com/priceline/design-system/pull/685